R/ui_functions.R

```diff
@@ -47,8 +47,11 @@ ui_hp_countries <- function(country = c("AGO", "CIV"),
   out$pop_in_poverty <- out$reporting_pop * out$headcount / pop_units
   out$reporting_pop  <- out$reporting_pop / pop_units
 
-  out <- out[, .(country_code, reporting_year, poverty_line, reporting_pop,
-                 pop_in_poverty)]
+  out <- out[, .(pcn_region_code, country_code, reporting_year,
+                 poverty_line, reporting_pop, pop_in_poverty)]
+  out <- out %>%
+    dplyr::rename(region_code = pcn_region_code)
+
```

Comment:
I think I removed (or was in the process of removing) `dplyr` dependency since this is quite a heavy package, and I want to minimize dependencies for packages used in PROD. Can you use base R or data.table instead? Thanks!

Reply:
Sure! Fully on-board with minimizing `dplyr` dependencies. It was still listed in Imports though, so we should probably to a fuller check as well. Can add a separate issue for that.

Reply:
Switched to data.table